src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationService.kt

```diff
@@ -214,7 +216,11 @@ class AdjudicationService(
           createRepairForAdjudicationIncident(incident = this, index + 1, repair)
         }
       }
-      .let { mapAdjudication(it.parties.findAdjudication(adjudicationNumber)) }
+      .let {
+        // created user needs refreshing from database - hence the flush and clear
+        adjudicationIncidentRepository.flush().also { entityManager.clear() }
+      }
```

Comment:
jump through hops so that the entity is re-read from database since the username is set by database/trigger not by JPA

Reply:
Why isn't this a flush and refresh instead of clear? Also wondering if you could use Hibernate's [Generated](https://docs.jboss.org/hibernate/orm/6.2/javadocs/org/hibernate/annotations/Generated.html) annotation instead?